content/docs/instrumenting/exporters.md

```diff
@@ -216,6 +216,9 @@ practices.
    * Go: [go-metrics instrumentation library](https://github.com/armon/go-metrics)
    * Go: [gokit](https://github.com/peterbourgon/gokit)
    * Go: [prombolt](https://github.com/mdlayher/prombolt)
+   * Java/JVM: [EclipseLink metrics exporter](https://github.com/VitaNuova/eclipselinkexporter)
```

Comment:
These are collectors, not exporters, as they are standalone exposing http themselves.

Reply:
Ok sorry, hopefully it's fine now.